src/current/_includes/common/basic-terms.md

```diff
@@ -34,24 +34,28 @@ Service accounts operate under a unified authorization model with organization u
 
 However, 'legacy service accounts' that were created before the updated authorization model was enabled for your cloud organization may have permissions assigned under the legacy model (like ADMIN, CREATE, EDIT, READ, DELETE). The legacy model for service accounts is now deprecated. It is recommended to update such service accounts with updated organization roles.
 
-To learn more, refer to [Manage Service Accounts]({{ link_prefix }})managing-access.html#manage-service-accounts)
+To learn more, refer to [Manage Service Accounts]({{ link_prefix }}managing-access.html#manage-service-accounts).
 
 #### CockroachDB {{ site.data.products.basic }} cluster
 
-A CockroachDB {{ site.data.products.cloud }} cluster deployed on request for a specific customer in *shared* network and compute infrastructure.
+A CockroachDB {{ site.data.products.cloud }} cluster with minimal operational features deployed in *shared* network and compute infrastructure.
 
-#### CockroachDB {{ site.data.products.dedicated }} cluster
+#### CockroachDB {{ site.data.products.standard }} cluster
 
-A CockroachDB {{ site.data.products.cloud }} cluster deployed on request for a specific customer, in a cloud provider's network and compute infrastructure *dedicated* to that customer, which can be distributed over multiple regions for added disaster-resilience. In addition to infrastructure isolation, dedicated clusters can be customized with advanced network, identity-management, and encryption-related security features.
+A CockroachDB {{ site.data.products.cloud }} cluster with full operational features and provisioned capacity, deployed in *shared* network and compute infrastructure.
+
+#### CockroachDB {{ site.data.products.advanced }} cluster
+
+A CockroachDB {{ site.data.products.cloud }} cluster with full operational capacity deployed in a cloud provider's network and compute infrastructure *dedicated* to each customer, which can be distributed over multiple regions for added disaster-resilience. In addition to infrastructure isolation, Advanced clusters can be customized with advanced security features for PCI DSS and HIPAA [compliance]({{ link_prefix }}compliance.html) at an additional cost.
```

Comment:
"which can be distributed over multiple regions for added disaster-resilience" in just Advanced Tier makes it sound like Standard  Tier doesn't support resiliency. I would also change disaster-resiliency to resiliency. 
Additional context: there are two aspects to providing resiliency - high availability (HA) and disaster recovery (DR)
HA maximizes uptime and ensures continuous and uninturrepted  access to data, even in presence of failures or disruption.
DR minimizes downtime and data loss and allows customers to recover from a major incident or disaster.

Reply:
agreed. also not a differentiator for advanced and can be deleted IMO.